Source/Common/EnvironmentLoader.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <cassert>
```

Comment:
Only include headers in here that you'll actually be using.
I don't believe any of these three need to be included here